lib/computer_player.rb

```diff
@@ -0,0 +1,10 @@
+class ComputerPlayer
```

Comment:
All of the classes with `Player` in the name are a great start for a duck type. Do all of your `Player` classes have the same methods with the same inputs and outputs? If not, what would it take to get them all in line? Also, if you were to duck type, this could be something like `EasyComputerPlayer` and the only thing that would need to change for medium and difficult would be the implementation of `get_move.

Reply:
That makes a lot of sense, I will take that into consideration on my next PR (medium computer).